### ISSUE TITLE
Bypass checks permissions are not working!

### ISSUE THREAD

**Issue by BadassOverlord** [NoCheatPlus/Issues#87]
Hello, I'm a server owner on my self-hosted server. None of the bypass permissions are working at all. I have to do /ncp exempt in order to bypass, but otherwise they don't work!

**Comment by asofold** [comment]
Could you provide more detailed information about the side conditions?
* Full text output of the 'ncp version' command.
* Permissions plugin (+ version).
* Which permissions did you give players and what did you test that didn't work?

**Comment by BadassOverlord** [comment]
Apologies for not including this in the first post. Anyway, here's the info:

---- Version information ----
[08:42:40 INFO]: #### Server ####
[08:42:40 INFO]: git-Spigot-44c59bf-8f0f4ed ~MC: 1.8.7~
[08:42:40 INFO]:   detected: 1.8.7
[08:42:40 INFO]: #### NoCheatPlus ####
[08:42:40 INFO]: Plugin: 3.13.7-RC-sMD5NET-b925
[08:42:40 INFO]: MCAccess: 1.8.4-1.8.8 / Spigot-CB-1.8_R3
[08:42:40 INFO]: Features:
[08:42:40 INFO]:   blocks: BlocksMC1_4 | BlocksMC1_5 | BlocksMC1_6_1 | BlocksMC1_7_2 | BlocksMC1_8
[08:42:40 INFO]:   checks: FastConsume | Gutenberg | AttackFrequency | FlyingFrequency | KeepAliveFrequency
[08:42:40 INFO]:   defaults: pvpKnockBackVelocity
[08:42:40 INFO]:   packet-listeners: UseEntityAdapter | MovingFlying | OutgoingPosition | KeepAliveAdapter | SoundDistance
[08:42:40 INFO]: Hooks: AllViolations~NCP~ 1.0 | Citizens2~cncp~ 2.0 | mcMMO~cncp~ 2.3
[08:42:40 INFO]: #### Related Plugins ####
[08:42:40 INFO]: CompatNoCheatPlus v6.6.2-RC-sMD5NET-b83 | ProtocolLib v3.6.4

bPermissions v2.12 Beta 10


Currently, I have given myself the following permissions related to NCP:

    - nocheatplus.admin
    - nocheatplus.bypass.denylogin
    - nocheatplus.checks
    - nocheatplus.mods.cjb
    - nocheatplus.mods.minecraftautomap
    - nocheatplus.mods.rei.cave
    - nocheatplus.mods.rei.radar
    - nocheatplus.mods.smartmoving
    - nocheatplus.mods.zombe
    - nocheatplus.shortcut.info
    - nocheatplus.shortcut.monitor

Basically what I tested was spamming in chat, spamming commands, and killaura with 100 enemies. It kicked me for all 3 things. The monitor permission is working though.

**Comment by asofold** [comment]
Are command protection features enabled (or hiding plugins)?

**Comment by BadassOverlord** [comment]
What do you mean by that? Is this something part of NCP or bPermissions?

**Comment by asofold** [comment]
It's NCP configuration, somewhere around the protection section in the configuration. If you haven't activated anything (hide plugins / nopermission or nocommand lists), then this is not related.
